packages/ilios-common/addon/components/daily-calendar.js

```diff
@@ -1,24 +1,21 @@
 import Component from '@glimmer/component';
 import { service } from '@ember/service';
-import { restartableTask, timeout } from 'ember-concurrency';
-import { action, set } from '@ember/object';
 import { sortBy } from 'ilios-common/utils/array-helpers';
+import { modifier } from 'ember-modifier';
 import { DateTime } from 'luxon';
 
 export default class DailyCalendarComponent extends Component {
   @service intl;
 
-  scrollView = restartableTask(async (calendarElement, [earliestHour]) => {
-    //waiting ensures that setHour has time to setup hour elements
-    await timeout(1);
-    // all of the hour elements are registered in the template as hour0, hour1, etc
-    let hourElement = this.hour6;
+  scrollView = modifier((element, [earliestHour]) => {
+    // all of the hour elements are registered in the template as hour-0, hour-1, etc
+    let hourElement = document.getElementsByClassName(`hour-6`)[0];
```

Comment:
suggesting to anchor this query to the passed-in container element instead of the document root note.
scoping this down reduces churn and prevents potential class-name collisions.

```suggestion
    let hourElement = element.getElementsByClassName(`hour-6`)[0];
```